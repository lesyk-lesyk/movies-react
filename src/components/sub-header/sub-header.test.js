import React from 'react';
import { SubHeader } from './index';
import renderer from 'react-test-renderer';

describe('SubHeader', () => {
  it('renders', () => {
    const tree = renderer
      .create(<SubHeader />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
