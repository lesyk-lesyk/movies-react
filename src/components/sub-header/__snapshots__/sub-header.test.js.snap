// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubHeader renders 1`] = `
<div
  className="sub-header"
>
  <div
    className="container sub-header-container"
  />
</div>
`;
